app/controllers/home.ts

```diff
@@ -4,11 +4,39 @@ import { action } from "@ember/object";
 import RouterService from "@ember/routing/router-service";
 import { service } from "@ember/service";
 import { tracked } from "@glimmer/tracking";
+import { ModelFrom } from "frontend-burgernabije-besluitendatabank/lib/type-utils";
+import HomeRoute from "frontend-burgernabije-besluitendatabank/routes/home";
 
 export default class HomeController extends Controller {
   @service declare router: RouterService;
   @service declare store: Store;
 
+  declare model: ModelFrom<HomeRoute>;
+  @tracked isLoadingMore = false;
+
+  @tracked loading = false;
+  @tracked errorMsg = "";
+
+  @action
+  async loadMore() {
+    if (this.model && !this.isLoadingMore) {
+      this.isLoadingMore = true;
+      const nextPage = this.model.currentPage + 1;
+      const agendaItems = await this.store.query(
+        "agenda-item",
+        this.model.getQuery({ page: nextPage })
```

Comment:
Shouldn't we add the filters here?

Reply:
I thought filtering should be agnostic to paginaton so I added it all to the route

Reply:
I agree with you pagination doesn't need to know about filter, but with your current implementation the loading nextPage doesn't apply filters. You can inject filters in `this.model.getQuery` from the route.

Reply:
Fixed!